### DISCUSSION TITLE
Error in running Bowtie2

### DISCUSSION THREAD

**Discussion by Rajeshwarira** [GuyTeichman/RNAlysis#24] [Q&A] [answered]
Hello Guy,
I have followed your instructions to install bowtie2 but still I'm not able to use it in RNAlysis.
I'm getting the error mentioned below :

Traceback (most recent call last):

File "rnalysis\utils\io.py", line 1479, in generate_base_call

File "rnalysis\utils\io.py", line 1420, in run_subprocess

File "subprocess.py", line 858, in init

File "subprocess.py", line 1311, in _execute_child

FileNotFoundError: [WinError 2] The system cannot find the file specified

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

File "rnalysis\gui\gui.py", line 3697, in finish_generic_job

File "rnalysis\gui\gui_widgets.py", line 329, in run

File "rnalysis\fastq.py", line 747, in bowtie2_align_paired_end

File "rnalysis\fastq.py", line 803, in _parse_bowtie2_misc_args

File "rnalysis\utils\io.py", line 1482, in generate_base_call

FileNotFoundError: RNAlysis could not find 'bowtie2'. Please ensure that your installation folder is correct, or add it to PATH.

I have been trying to figure out this from the past two days. Still I didn't find any solution.

Can you please help me with this?

![IMG20230523153036](https://github.com/GuyTeichman/RNAlysis/assets/128403613/58c58ba5-e1f3-440e-85a9-7cc312bda211)

Thank you
Regards

**Comment by Rajeshwarira** [comment]
Hello Guy,
Please do help me with this.
I'm really in need of it.
Thank you

**Comment by GuyTeichman** [comment]
Hi,
What is the value you set for the parameter "bowtie2_installation_folder"?

**Comment by Rajeshwarira** [comment]
I didn't set any I went with the default.

**Comment by Rajeshwarira** [comment]
Hello Guy,
Good noon,
Can you please guide me what parameters am I suppose to set ??

**Comment by Rajeshwarira** [comment]
The parameters now is
@echo off

perl %~dp0/bowtie2 %*

**Comment by GuyTeichman** [comment]
Hi,
please try setting the 'bowtie2_installation_folder' to point to the folder in which you installed bowtie2.

**Comment by Rajeshwarira** [comment]
Hello Guy,
I'm really sorry to bother you a lot. But I'm not sure which is
installation folder.

**Comment by Rajeshwarira** [comment]
Hello Guy,
Can you please guide me with this step by step??
Thank you

**Comment by GuyTeichman** [comment, marked as answer]
You have downloaded and installed the tool 'bowtie2'. You installed it at a specific location on your computer - this is the "installation folder". 
You should set the 'bowtie2_installation_folder' parameter in RNAlysis, so it points to the folder in which you installed bowtie2.
